### DISCUSSION TITLE
Problem with safe exam browser Windows both version

### DISCUSSION THREAD

**Discussion by ari1101** [SafeExamBrowser/seb-win-refactoring#352] [Q&A]
Good afternoon greetings

I have a problem with the safe Exam Browser, the configuration file opens it with the program notepad, on a computer of one of our students, we tried to uninstall the safe exam browser and reinstall it, and it did not work.
Another method we used was to search from the C folder of the local disk for the safe examination browser program, but that did not work for us either, so we were left with the doubt that we could continue doing.
Note: It would be for the Windows 10 operating system and the safe exam browser version 2.4.1, we tried both versions for the Windows safe exam browser but it gives us the same error.

We wait for your comments

**Comment by dbuechel** [comment]
We'd need a bit more details regarding what exactly "did not work": Was there an error message, and if yes, could you please share it here? In general, you should never manipulate the installation folder of an application on your own (unless you know exactly what you're doing) and rather use the application control panel of Windows to uninstall software.

Have you tried to upgrade SEB to the latest version? The setup bundle of SEB 3.x should automatically and correctly uninstall SEB 2.x and then install SEB 3.x and all its runtime dependencies.

**Reply by ari1101** [reply]
![MicrosoftTeams-image (2)](https://user-images.githubusercontent.com/91102971/154381519-acc2ee48-1b04-4a8d-8f2f-1991f38e3c7e.png)

**Reply by ari1101** [reply]
Good night.

Attached image referring to the error generated on Safe Exam Browser.
Thanks.

**Reply by ari1101** [reply]
And then this week with Safe Exam Browser 2.4.1 on Windows 10, the following error appeared in the attached video with multiple student computers.
We do not know what it can be, I am attentive to your response,
Thanks

**Reply by dbuechel** [reply]
Please update to version 3.x whenever possible. If notepad is being used to open a SEB configuration file, then the file association for `.seb` files on that computer is wrong / corrupted and would need to be fixed (in case it won't be fixed with the upgrade to SEB 3.x).

**Reply by ari1101** [reply]
good evening
Thank you for your answer, we will take it into account to update to the new version

**Comment by ari1101** [comment]
https://user-images.githubusercontent.com/91102971/154382348-df90acd7-dbb8-40ee-b416-a30cf93cabfd.mp4

**Reply by dbuechel** [reply]
This looks like an issue with Cisco WebEx, where the meeting application would reactivate the Windows desktop (see [#333](https://github.com/SafeExamBrowser/seb-win-refactoring/discussions/333#discussioncomment-2155423)). That issue will be fixed with the next update, according to the comment linked before.

**Reply by ari1101** [reply]
good evening
Thank you for your answer, we will take it into account to update to the new version
